### PR TITLE
Fixing small html issues in flashing patterns docs

### PR DESCRIPTION
**Pull request by jackwardell**
Fixing some simple syntax in flashing patterns.

The examples in https://flask.palletsprojects.com/en/1.1.x/patterns/flashing/ are shown as:
`<ul class=flashes>`
but rather should be:
`<ul class="flashes">`

More examples of this throughout the page. Just fixing it up.

Also touching up some HTML by removing unnecessary p tag and swapping dl tags for input/label tags as they're better for screen readers and improve accessability.

### CODE REVIEW

**Comment by jackwardell** [comment]
should I have raised an issue first?

File: docs/patterns/flashing.rst [hunk 1]
```diff
@@ -48,7 +48,7 @@ And here is the :file:`layout.html` template which does the magic:
    <title>My Application</title>
    {% with messages = get_flashed_messages() %}
      {% if messages %}
-       <ul class=flashes>
+       <ul class="flashes">
```
Comment (ThiefMaster) [inline docs/patterns/flashing.rst:51]: quotes are optional

File: docs/patterns/flashing.rst [hunk 2]
```diff
@@ -110,7 +110,7 @@ categories.  The loop looks slightly different in that situation then:
 
 .. sourcecode:: html+jinja
 
-   {% with messages = get_flashed_messages(with_categories=true) %}
+   {% with messages = get_flashed_messages(with_categories=True) %}
```
Comment (ThiefMaster) [inline docs/patterns/flashing.rst:113]: lowercase versions of true/false/none are the default in jinja, the python-cased variants are just there as an alias.

Comment (jackwardell) [reply docs/patterns/flashing.rst:113]: thanks for this, I didn't realise! True had always worked for me and I never realised lowercase was the default

File: docs/patterns/flashing.rst (hunk 1 above)
Comment (jackwardell) [reply docs/patterns/flashing.rst:51]: true, and it's probably my pedanticism, but it's inconsistent across the example and I would argue double quotes are preferred, but fair enough

**Pull request closed without merging**